dev-doc/RegenerateLanguages.md

```diff
@@ -0,0 +1,45 @@
+# Regenerate Languages
+
+This document describes things that need to be done when regenerate the test- and production languages.
```

Comment:
s/things that need to be done when/the necessary steps to/

Reply:
Fixed.